### DISCUSSION TITLE
How do I construct a population with liver damage in PK-Sim?

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1680] [How-Tos]
I first modified the corresponding physiological parameters in the individual, and then in the population, I scaled the corresponding individual to the population. However, according to the final simulation results, there is a large deviation between the population and the individual simulation values. How do I scale an individual with modified physiological parameters to a population in PK-Sim?

**Comment by tobiasK2001** [comment]
Hi, generally the differences see in Population simulations and and individual simulation can have several reasons. Hrad to comment without seeing the project . But just some quick thoughts, to scale an individual you cane scale it with the inbuild scale function: 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/61178dde-a48e-42c1-8679-28d95e591d73)

However to investigate the differences between your population simulation and your individual simulation a bit deeper it might be a good idea to extract an typical (our an extreme) Individual from your created population 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/ed424e1c-8a5e-4906-98cc-40aafdaf2021)

you can chose the individual in the next step by giving the individual ID.
Than build agan a individual simulation with the extracted individual and compare the original individual Simulation with the mean individual and see what differences are there.

Hope that might help you.

Best, Tobias
